sql/core/src/test/scala/org/apache/spark/sql/execution/command/ShowTablesSuiteBase.scala

```diff
@@ -57,6 +58,13 @@ trait ShowTablesSuiteBase extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("show table in a not existing namespace") {
+    val msg = intercept[NoSuchNamespaceException] {
```

Comment:
@cloud-fan and @MaxGekk .
This means the behavior change for both v1 and v2, doesn't it?
- `v1`: `NoSuchDatabaseException` -> `NoSuchNamespaceException`
- `v2`: empty list -> `NoSuchNamespaceException`

Reply:
I didn't touch v1, it behaves as before. `NoSuchDatabaseException` is a sub-class of `NoSuchNamespaceException`. or do you mean the change in the test?

Reply:
Got it~, @MaxGekk .